pkgs/stdenv/darwin/stdenv-bootstrap-tools.nix

```diff
@@ -292,6 +292,6 @@ stdenv.mkDerivation (finalAttrs: {
   };
 
   meta = {
-    maintainers = [ lib.maintainers.copumpkin ];
+    maintainers = [ ];
```

Comment:
```suggestion
    maintainers = lib.teams.darwin.members;
```

Reply:
Seems to be technically unrelated, but added :)